assets/less/sidebar.less

```diff
@@ -242,7 +242,7 @@
         margin-left: 10px;
       }
 
-      a.expand {
+      a.expand > span {
```

Comment:
Maybe include the `.icon-expand` class for clarity?

Reply:
Good idea!